Documentation/cli/README.md

```diff
@@ -581,6 +582,14 @@ Print package variables.
 If regex is specified only package variables with a name matching it will be returned. If -v is specified more information about each package variable will be shown.
 
 
+## watch
+Set watchpoint.
+	
+	watch [-r|-w|-rw] <expr>
+
+The same expression language used by 'print' is understood by watch.
```

Comment:
Expand on this as well for the user, clarify "expression language" and maybe suggest that the user can type `help print` for more info.